### ISSUE TITLE
After running pod install it removing the react and also after unlinking the packages podfile not showing any third party packages in latest react-native version(0.60.0)

### ISSUE THREAD

**Issue by nihp** [facebook/react-native#27358]
I am upgrading the react-native versions. 

I have run pod install before unlinking the packages and then perform unlink and then I have run pod install.

After this, it shows react missing. So again I added all the removed packages in podfile and run pod install.

Now I am facing the react-missing and React/RCTBundleURLProvider.h' file not found error. 

I have tried with clean pod, node_modules and derived datas but nothing worked for me.

Any solution?
Details:

```
System:
    OS: macOS Mojave 10.14.2
    CPU: (4) x64 Intel(R) Core(TM) i5-5350U CPU @ 1.80GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 9.4.0 - ~/.nvm/versions/node/v9.4.0/bin/node
    npm: 5.6.0 - ~/.nvm/versions/node/v9.4.0/bin/npm
  SDKs:
    iOS SDK:
      Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
  IDEs:
    Xcode: 10.1/10B61 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.8.6 => 16.8.6 
    react-native: 0.60.0 => 0.60.0 
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-git-upgrade: 0.2.7
```

**Comment by nihp** [comment]
I have updated my comment

**Comment by dulmandakh** [comment]
Please make sure that you follow https://react-native-community.github.io/upgrade-helper/ thoroughly, and please note that 0.60 features auto-linking feature.

**Comment by nihp** [comment]
Thanks I already saw this.

I have a question @dulmandakh My Podfile target has a lot of third party packages before after run pod install and  I have unlinked all packages and the Podfile not showing the third party packages. Did you know the reason?

Now it is empty. If I add pod and paths which I had before it shows the error as need to unlink <packages>

If I remove and run the app it not shows this error.

Also I am facing React/{.h} not found error for more than 2 weeks. I can'table to fix that. I tried many solution nothing get fixed
